toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -1,8 +1,8 @@
-# Mariner Image Customizer ISO Support
+# Azure Linux Image Customizer ISO Support
 
 ## Overview
 
-Given a full disk image, the Mariner Image Customizer (MIC) can generate a
+Given a full disk image, the Azure Linux Image Customizer (MIC) can generate a
```

Comment:
And here.  MIC-> ALIC?

Reply:
Checked with Chris, ALIC is not good acronym. Decided to drop use of acronym everywhere